src/app/layout/app-info/app-info.component.scss

```diff
@@ -1,13 +1,29 @@
 @import '../../../injects';
 
 .app-info{
+
+	&::-webkit-scrollbar-track {
+	    background-color: white;;
+    }
+    &::-webkit-scrollbar
+    {
+        width: 6px;
+        background-color: #F5F5F5;
+    }
+
+    &::-webkit-scrollbar-thumb
+    {
+        background-color: rgb(237, 241, 247);
+    }
+	
 	background:$appinfo;
 	@include transition((width 0.3s,right .5s));
 	position: fixed;
 	right:-300px;
 	top:0px;
 	bottom:0;
 	width:300px;
+	overflow-x: hidden;
 	@media (min-width: $screen-1366){
```

Comment:
There is a big issue with this line. If you use double finger on pad, you can still scroll to left-right and that's terrible.